src/nupic/os/Env.cpp

```diff
@@ -126,6 +126,8 @@ char ** Env::environ_ = nullptr;
 
 #if defined(NTA_OS_DARWIN)
 #include <crt_externs.h>
+#elif defined(NTA_OS_WINDOWS)
+_CRTIMP extern char **_environ;
```

Comment:
Please create an issue to track this and add a comment here with a link to the issue.